src/java.base/share/classes/java/lang/foreign/VaList.java

```diff
@@ -52,9 +52,13 @@
  * As such, this interface only supports reading {@code int}, {@code double},
  * and any other type that fits into a {@code long}.
  * <h2 id="safety">Safety considerations</h2>
- * The behavior of any attempts to access a value in the variable argument list, whether through one of the {@code nextVarg} overloads
- * or {@link #skip(MemoryLayout...)}, using a memory layout other than the layout of the accessed value, or any subsequent
- * accesses to the same variable argument list, is undefined.
+ * Accessing a value through a variable argument list using the wrong memory layout will result in undefined behavior.
```

Comment:
Note that I went with "through" here, instead of "in", since we describe a va list as a "stateful cursor used to iterate over a set of arguments" in the section before this.

Reply:
I think we can condense this:

```
 For instance, if a variable argument list currently points at a C {@code int} value, then accessing it using
 * {@link #nextVarg(ValueLayout.OfLong)} is illegal. Similarly, accessing the variable argument list with  {@link #skip(MemoryLayout...)}, and providing a layout other than {@link ValueLayout.OfInt} is illegal. Any such illegal accesses might not be detected by the implementation, and can corrupt the variable argument list, so that the behavior of subsequent accesses is also undefined.
 ```

Reply:
Done